File: app/src/main/java/sbs/pros/parking/landlord_main/LandlordMenu.java
package sbs.pros.parking.landlord_main;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.MenuItem;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.viewpager.widget.ViewPager;

import com.google.android.material.bottomnavigation.BottomNavigationView;
import com.google.android.material.navigation.NavigationBarItemView;
import com.google.android.material.tabs.TabLayout;

import sbs.pros.parking.R;
import sbs.pros.parking.landlord_main.ui.home.HomeFragment;

public class LandlordMenu extends Fragment {

    private BottomNavigationView bottomNavigationView;
    private InFragment inFragment = new InFragment();
    private ParkedFragment parkedFragment = new ParkedFragment();
    private OutFragment outFragment = new OutFragment();



    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_landlord_auth, container, false);

        bottomNavigationView = bottomNavigationView.findViewById(R.id.bottom_menu);

        bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                Fragment selectedFragment = null;

                switch (item.getItemId()){

                }


            }
        });



        return view;
    }


}

File: app/src/main/java/sbs/pros/parking/landlord_main/LandlordMenu.kt
package sbs.pros.parking.landlord_main

import com.google.android.material.bottomnavigation.BottomNavigationView
import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import kotlinx.android.synthetic.main.fragment_landlord_menu.*
import sbs.pros.parking.R
import sbs.pros.parking.databinding.FragmentLandlordMenuBinding
import sbs.pros.parking.databinding.FragmentMapBinding
import sbs.pros.parking.utils.viewLifecycleLazy

class LandlordMenu : Fragment(R.layout.fragment_landlord_menu) {

    private val inFragment = InFragment()
    private val parkedFragment = ParkedFragment()
    private val outFragment = OutFragment()


    private val binding by viewLifecycleLazy { FragmentLandlordMenuBinding.bind( requireView()) }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        replaceFragment(inFragment)

        bottom_menu.setOnNavigationItemSelectedListener {
            when (it.itemId){
                R.id.`in` -> replaceFragment(inFragment)
                R.id.out -> replaceFragment(outFragment)
                R.id.parked -> replaceFragment(parkedFragment)
            }
            true
        }
    }


    private fun replaceFragment(fragment: Fragment){
        if (fragment != null){


            val transaction = parentFragmentManager.beginTransaction()
            transaction.replace(R.id.fragment_container, fragment)
            transaction.commit()
        }
    }
}